rtsp.go

```diff
@@ -148,8 +149,37 @@ func (rc *rtspCamera) reconnectClient() (err error) {
 		return err
 	}
 
-	// find the H264 media and format
+	codecInfo := getStreamInfo(rc.u.String())
+	rc.logger.Infof("codec info: %v", codecInfo)
+
+	switch codecInfo {
+	case "h264":
+		rc.logger.Infof("setting up H264")
+		err = rc.initH264(tracks, baseURL)
+	case "h265":
+		rc.logger.Infof("setting up H265")
+		err = rc.initH265(tracks, baseURL)
+	default:
+		return errors.New("codec not supported")
```

Comment:
add the name of the codec name that's not supported, so we have information if a client ever hits this

Reply:
Good call - added codec name to error statement here.